templates/partials/preview-question.html

```diff
@@ -97,19 +104,20 @@ <h3 class="ons-u-mt-m ons-u-fs-r--b">{{ question.title }}</h3>
         {% set instruction = answer.instruction %}
         <p>{{- instruction | safe -}}</p>
       {% endif %}
-
+      {% if answer.label %}
+        {% set label = answer.label %}
+        <p>{{- label | safe -}}</p>
+      {% endif %}
+      {% if answer.max_length %}
+          <p>{{ answer.max_length }}{{ _(" characters can be added.") }}</p>
```

Comment:
This has to be one string, this wouldn't work with translation. i.e `{max_characters} characters can be added.` etc.

Reply:
Added a fix.